.github/workflows/ci.yml

```
@@ -13,7 +13,7 @@ jobs:
     continue-on-error: ${{ matrix.continue-on-error }}
     strategy:
       matrix:
-        go-version: [ "1.22" ]
+        go-version: [ "1.25" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)